tests/Testcontainers.Weaviate.Tests/WeaviateContainerTest.cs

```diff
@@ -0,0 +1,28 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using DotNet.Testcontainers.Commons;
+using Xunit;
+
+namespace Testcontainers.Weaviate;
+
+public sealed class PostgreSqlContainerTest : IAsyncLifetime
```

Comment:
This looks like a copy paste error. I am guessing what you intended to name this was `WeaviateContainerTest`

Reply:
Thanks, fixed.